packages/design-system/src/components/svgIcon/svgIcon.tsx

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+interface SVGIconProps {
+  glyph: string;
+  viewBox: string;
+  className?: string;
+  title?: string;
+}
+
+export default function SVGIcon({
```

Comment:
IIUC this is only really needed in the webpack config and not something that‘s imported elsewhere. So not really something for this package. Can this just live in the runtime generator or so?